lib/vet360/models/telephone.rb

```diff
@@ -46,6 +51,20 @@ class Telephone < Base
         presence: true,
         inclusion: { in: PHONE_TYPES }
       )
+
+      def number
+        area_code + phone_number
+      end
+
+      def details
+        {
+          id: id,
+          number: number,
+          extension: extension,
+          country_code: country_code,
+          phone_type: phone_type
+        }
+      end
```

Comment:
@bshyong @ncksllvn - anything else you'd like to see returned for a phone?

Reply:
💯